patches/components-policy-resources-policy_templates.json.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/components/policy/resources/policy_templates.json b/components/policy/resources/policy_templates.json
+index 1aefb537e31087d645307d6bac4ef3448cda3e40..33a3a603447954f7ff647b65ce5daa3c5eb11309 100644
+--- a/components/policy/resources/policy_templates.json
++++ b/components/policy/resources/policy_templates.json
+@@ -17623,6 +17623,21 @@
+ 
+       See https://www.chromestatus.com/feature/4664843055398912 .''',
+     },
++    {
```

Comment:
we need to find a way to do this, or at the very least it needs to be compressed down to one line

Reply:
I did see some comments that this is not a pure json file and supports some additional features so maybe we can utilize those or maybe parse this template, make changes to it and output again? Or maybe we can use a python include and insert directly into the parsed data?

Reply:
for instance maybe in generate_policy_source we can merge the original file with one we add here `template_file_contents = _LoadJSONFile(template_file_name)`?

Reply:
Compressed to one line.

Reply:
@bridiver `policy_source_helper.py` is introduced and it adds our policies. PTAL again.